css/css-contain/content-visibility/content-visibility-with-popover-top-layer-006.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<meta charset="utf8">
+<title>CSS Content Visibility: offscreen c-v auto content is relevant when in top layer</title>
+<link rel="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-contain/#content-visibility">
+<meta name="assert" content="offscreen c-v auto content is relevant when in top layer">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<style>
+#inner {
+    content-visibility: auto;
+    contain-intrinsic-size: 100px 100px;
+}
+</style>
+
+<div popover id="popover">
+    <div id="spacer" style="height: 100000px;"></div>
+    <div id="inner">
+        <div style="height: 200px; width: 200px;">content</div>
+    </div>
+</div>
+
+<script>
+test(() => {
+    popover.showPopover();
+
+    // Even though the element with `content-visibility: auto` has an ancestor
+    // in the top layer, this element is not in the top layer list (although
+    // it's in the top layer stacking context). This means it should not be
+    // relevant to the user, because it is nevertheless not onscreen.
```

Comment:
```suggestion
    // relevant to the user, because it is not onscreen.
```

nit: The use of two negative words confused me a bit here

Reply:
Fixed.